core/src/main/java/io/undertow/server/session/InMemorySessionManager.java

```diff
@@ -413,17 +413,17 @@ private SessionImpl(final InMemorySessionManager sessionManager, final String se
             this.worker = worker;
             this.evictionToken = evictionToken;
             creationTime = lastAccessed = System.currentTimeMillis();
-            this.maxInactiveInterval = maxInactiveInterval;
+            this.setMaxInactiveInterval(maxInactiveInterval);
```

Comment:
Initially I defined another var to store this, but this might introduce race? Removing unnecessary multiplication in bumpTimeout should be enough.

Reply:
agreed